generator/rooms.py

```diff
@@ -0,0 +1,30 @@
+import random
+
+
+def generateRoomInformation():
+    # Name
+    name = generateName()
+
+    # Price
+    price = generatePrice()
+
+    # Assign Room Information:
+    roomInformation = {'name': name,
+                       'price': price
+                       }
+
+    return roomInformation
+
+
+def generateName():
+    name = 'room#'
+
+    name += str(random.randint(0, 99))
```

Comment:
There could be collisions. So, you could have a few different rooms with the same name. As Ican see in usage code, it could bring some confusion. From my perspective u'd need some storage with already taken numbers as following:
u calculate random value and check whether its taken or not. If taken - try next value until u have new unique value.